frontend_tests/node_tests/upload.js

```diff
@@ -46,6 +46,7 @@ run_test('get_item', () => {
     assert.equal(upload.get_item("file_input_identifier", {mode: "compose"}), "#file_input");
     assert.equal(upload.get_item("source", {mode: "compose"}), "compose-file-input");
     assert.equal(upload.get_item("drag_drop_container", {mode: "compose"}), $('#compose'));
+    assert.equal(upload.get_item("drag_drop_container_img", {mode: "compose"}), $("#main_div"));
```

Comment:
I would use a better name instead of `drag_drop_container_img`. You can technically drag files other than the image as well. Also, make the variable name convey that the container is not restricted to only the compose box.